samples/cpp/build_samples.sh

```diff
@@ -14,8 +14,7 @@ usage() {
     exit 1
 }
 
-samples_type="$(basename "$(dirname "$(realpath "${BASH_SOURCE[0]}")")")"
-build_dir="$HOME/openvino_${samples_type}_samples_build"
+build_dir="$HOME/openvino_cpp_samples_build"
```

Comment:
the same script is used for C samples build.

Reply:
What is the scenario? Is the script copied to `c` folder before running?

Reply:
In OpenVINO installer:
- samples/c/build_samples.sh
- samples/cpp/build_samples.sh

Reply:
Updated